src/components/Match/matchColumns.jsx

```diff
@@ -541,11 +528,7 @@ export const inflictorsColumns = [{
     field: 'damage_inflictor',
     displayFn: (row, col, field) => (field ? Object.keys(field)
       .sort((a, b) => field[b] - field[a])
-      .map((k, i) => inflictorWithValue({
-        inflictor: k,
-        value: abbreviateNumber(field[k]),
-        key: i,
-      })) : ''),
+      .map(k => inflictorWithValue(k, abbreviateNumber(field[k]))) : ''),
```

Comment:
I agree with Howard, it's easier to use. My only concern would be creating objects just to pass data if we were dealing with lots of rows could actually be a performance issue potentially.